net-libs/mbedtls/mbedtls-3.5.2.recipe

```diff
@@ -0,0 +1,162 @@
+SUMMARY="Open source, portable, easy to use, readable and flexible SSL library"
+DESCRIPTION="mbed TLS (formerly known as PolarSSL) makes it trivially easy for \
+developers to include cryptographic and SSL/TLS capabilities in their (embedded) \
+products, facilitating this functionality with a minimal coding footprint."
+HOMEPAGE="https://tls.mbed.org/"
+COPYRIGHT="2006-2018 ARM Limited"
+LICENSE="Apache v2"
+REVISION="3"
+SOURCE_URI="https://github.com/Mbed-TLS/mbedtls/archive/refs/tags/v$portVersion.tar.gz"
+CHECKSUM_SHA256="35890edf1a2c7a7e29eac3118d43302c3e1173e0df0ebaf5db56126dabe5bb05"
+SOURCE_DIR="mbedtls-$portVersion"
+PATCHES="mbedtls-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersionmbedCrypto="$portVersion"
+libVersionembedTls="$portVersion"
+libVersionembedX509="$portVersion"
+libVersionembedCryptoCompat="$libVersionmbedCrypto compat >= 3.5.2"
+libVersionembedTlsCompat="$libVersionembedTls compat >= 3.5.2"
+libVersionembedX509Compat="$libVersionembedX509 compat >= 3.5.2"
```

Comment:
Seeing these all use $portVersion compat >= 3 this could be reduced to one statement:
```suggestion
libVersion="$portVersion"
libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
```